### ISSUE TITLE
MBTF incorrectly exports Dashboard Filters

### ISSUE THREAD

**Issue by cabarria** [flovouin/terraform-provider-metabase#58]
MBTF should correctly export the definitions of dashboard filters. The following table shows the source definition and what mbtf exports.

![image](https://github.com/flovouin/terraform-provider-metabase/assets/18726993/4ad52e94-42ab-4b84-a784-0ee75627a012)

**Comment by flovouin** [comment]
Hi,
Thanks for the report. I'm not sure what the table represents. It looks like filter names from a specific dashboard you're working on.
I'm guessing filters that should be dropdown list are exported as simple text boxes. Is that right? Are there other cases? Can you provide a minimal example of Terraform configuration that doesn't provide the expected filter type?

**Comment by cabarria** [comment]
That's correct. For example, we may have a dropdown filter with a custom list of values that translates to a dropdown list with an empty list of values. 
[mb-gen-dashboard-aws-waf-threat-intelligence.tf.json](https://github.com/user-attachments/files/15740909/mb-gen-dashboard-aws-waf-threat-intelligence.tf.json)
